packages/shortest/src/ai/tools/index.ts

```diff
@@ -30,6 +30,42 @@ export const AITools = [
       required: ["action", "username", "password"],
     },
   },
+  {
+    name: "check_email",
+    description: "Fetch, decode and render received email in browser",
+    input_schema: {
+      type: "object",
+      properties: {
+        action: {
+          type: "string",
+          enum: ["check_email"],
+          description: "Render the received email in a new tab"
```

Comment:
Not familiar yet on why the action needs to be repeated, but can it have the same description, for consistency?

```suggestion
          description: "View received email in new browser tab"
```

Reply:
We don't need to have this again actually but we can use it in case we'd like to give AI more context. Should I remove this you think?

Reply:
I added Sahil's recommendation for Action description and yours for tool description. I think this would give AI more context and help it to be more consistent.